api/actions/__test__/createWorkItem.spec.js

```diff
@@ -164,8 +165,71 @@ const reqWithIssueTypeLabels = {
     }
 };
 
+const reqWithGusTitlePrefix = {
+    body: {
+        issue: {
+            url: 'github/git2gus-test/#30',
+            title: 'new issue',
+            body: 'some description',
+            number: 30,
+            labels: [{ name: 'BUG P1' }]
+        },
+        label: 'BUG P1',
+        repository: {
+            name: 'git2gus-test',
+            owner: {
+                login: 'john'
+            }
+        }
+    },
+    git2gus: {
+        config: {
+            productTag: 'abcd1234',
+            defaultBuild: '218',
+            gusTitlePrefix: '[Some Prefix Text]'
+        }
+    },
+    octokitClient: {
+        issues: {
+            createComment: jest.fn()
+        }
+    }
+};
+
+const reqWithCommentsSuppressed = {
+    body: {
+        issue: {
+            url: 'github/git2gus-test/#30',
+            title: 'new issue',
+            body: 'some description',
+            number: 30,
+            labels: [{ name: 'BUG P1' }]
+        },
+        label: 'BUG P1',
+        repository: {
+            name: 'git2gus-test',
+            owner: {
+                login: 'john'
+            }
+        }
+    },
+    git2gus: {
+        config: {
+            productTag: 'abcd1234',
+            defaultBuild: '218',
+            suppressGithubComments: true
+        }
+    },
+    octokitClient: {
+        issues: {
+            createComment: jest.fn(),
+            updateDescription: jest.fn()
+        }
+    }
+};
+
 describe('createGusItem action', () => {
-    it('should call queue push with the right values', async () => {
```

Comment:
Felt this was a redundant test. Changed it to make sure we process we strip down markdown to GUS formatting.